base_classes/NXobject.nxdl.xml

```diff
@@ -64,6 +64,23 @@
             Boolean mask of FIELDNAME values. The value is masked if set to 1.
         </doc>
     </field>
+	<field name="FIELDNAME" type="NX_TYPE" minOccurs="0">
+        <doc>
+            This is a generic field that defines the attributes that all fields can have.
+        </doc>
+		<attribute name="target" type="NX_CHAR">
+		    <doc>
+				This attribute can be used to describe that this field is connected to another field.
+				For links, this must be set to the path of the target of the link.
+			</doc>
+		</attribute>
+    </field>
+	<attribute name="target" type="NX_CHAR">
+		<doc>
+			This attribute can be used to describe that this group is connected to another group.
```

Comment:
I see, now the nesting, 
nevertheless for the docstring

"to another {field, group}" add "- its target -"